### PR TITLE
feat/cp 111 dashboard style tweak

### PR DESCRIPTION
**Pull request by NorrisWu0**
https://beequant.atlassian.net/browse/CP-111

---

## What happens here?

as requested, dashboard should be identical to what the old app has. 

except the following
- could not reproduce the feature where you can click on a collapsed sidebar item to show/hide options
https://github.com/user-attachments/assets/147ac64d-bf66-4640-a14b-2cdfec50e180

- could not reproduce the feature where multiple chevron could appear at the same time
image

- replace the hamburger menu icon with an actual icon (it was using an image)
image

- when the sidebar is expanded, opening any accordion will shift the menu one pixel up (?????), could not reproduce either and don't quite understand what the meaning is behind this behavior.
image

## How do I know this is working?

run both app and switch tab

## Any notes?
also removed the duplicated `favicon.icon` in the app directory and suppressed the hydration warning re extra `class` attribute on html tag.

### CODE REVIEW

File: apps/client/src/module/dashboard/layout/account-menu.tsx [hunk 1]
```diff
@@ -28,62 +31,83 @@ export function AccountMenu() {
...
         )}
         {!user && "Loading..."}
       </button>
 
       <menu
         className={clsx(
-          "bg-primary-50 dark:bg-primary-500 absolute right-0 top-full grid w-60 transition-[grid-template-rows_padding] duration-300",
-          showDropdown ? "grid-rows-[1fr] py-2" : "grid-rows-[0fr] py-0",
+          "bg-primary-50 dark:bg-primary-900 absolute right-0 top-full z-[101] grid w-[210px] shadow-[0_2px_15px_0_rgba(0,0,0,0.05)] transition-[grid-template-rows_padding] duration-300",
+          showDropdown ? "grid-rows-[1fr] py-[15px]" : "grid-rows-[0fr] py-0",
```
Comment (s3713434) [inline apps/client/src/module/dashboard/layout/account-menu.tsx]: :3 About these values such as `[15px]` , Do you think we should also create some standard (???) in our config files? Or just leave it here?

**Review by s3713434 — COMMENTED** [review]
272d3854c979ad6346431146db5f089f
Looks good! Thanks for your working!

For dashboard just one thing:
image
image

And another thing that's not in dashboard:

https://github.com/user-attachments/assets/f357e768-c3e9-44de-b01f-0f2863b6da77

I don't know why the hover effect on the login button is not working again.

Logo LGTM, but I think it's best to wait for @Chris-Wang  to confirm it.

**Review by Chris-Wang — CHANGES_REQUESTED** [review]
Shout out to Norris! This is one of the most exciting work I see!

Close to 95%+ the same as the original. And you did the top bar as well!!

The only point I want to say:
Should we make the root/parent font un-bold like the original when fold? Coz it will become un-bold when un-fold; and individual drawer font is un-bold either;
New:
image
Original:
image


unfold:
image
other drawer font:
image

File: apps/client/src/module/dashboard/layout/account-menu.tsx (hunk 1 above)
Comment (Chris-Wang) [reply apps/client/src/module/dashboard/layout/account-menu.tsx]: I think the burger button icon is fine, it's mentioned in the note

Comment (Chris-Wang) [reply apps/client/src/module/dashboard/layout/account-menu.tsx]: yeah, I can confirm the button anima on login page is not working, is that happened before this PR?

**Comment by NorrisWu0** [comment]
> 272d3854c979ad6346431146db5f089f
> Looks good! Thanks for your working!
> 
> For dashboard just one thing:
> image
> image
> 
> And another thing that's not in dashboard:
> 
> https://github.com/user-attachments/assets/f357e768-c3e9-44de-b01f-0f2863b6da77
> 
> I don't know why the hover effect on the login button is not working again.
> 
> Logo LGTM, but I think it's best to wait for @Chris-Wang  to confirm it.

Read description re menu icon. 

I can check what's happening to the button later.

**Comment by NorrisWu0** [comment]
> Shout out to Norris! This is one of the most exciting work I see!
> 
> Close to 95%+ the same as the original. And you did the top bar as well!!
> 
> The only point I want to say:
> Should we make the root/parent font un-bold like the original when fold? Coz it will become un-bold when un-fold; and individual drawer font is un-bold either;
> New:
> image
> Original:
> image
> 
> 
> unfold:
> image
> other drawer font:
> image
> 

So just the font and that padding when the sidebar is collapsed yeah?

**Comment by Chris-Wang** [comment]
Shout out to Norris! This is one of the most exciting work I see

> > Shout out to Norris! This is one of the most exciting work I see!
> > Close to 95%+ the same as the original. And you did the top bar as well!!
> > The only point I want to say:
> > Should we make the root/parent font un-bold like the original when fold? Coz it will become un-bold when un-fold; and individual drawer font is un-bold either;
> > New:
> > image
> > Original:
> > image
> > unfold:
> > image
> > other drawer font:
> > image
> 
> So just the font and that padding when the sidebar is collapsed yeah?

yes, you get my point!

The button is not working under dev neither, I will make a ticket for that

**Comment by Chris-Wang** [comment]
created, button issue could be resolved in another ticket:
https://beequant.atlassian.net/browse/CP-112

**Comment by NorrisWu0** [comment]
done 
- https://beequant.atlassian.net/browse/CP-112
  - tweak accent color palette (swap #70bbfd (now 400) and #57abf4 (now 500), tho #57abf4 sits closer to 300, but in reality, it looks slightly darker than #70bbfd. 🤷 programmer eyes.
- fix the padding on the sidebar menu items, they should now all be aligned with a padding of 16 px.

@Chris-Wang @s3713434 

re. standard around [15px] etc, I guess this is the most complicated bit, it's hard to create a rule around this without knowing all the parameters. I reckon we can just stick to using arbitrary value for now, until we got better clarity on the scale of task here.

**Comment by Chris-Wang** [comment]
I found now the font size on the sidebar is different from what it used to be...
now:
image
on platform_app:
image

Can we fix it here?

**Comment by Chris-Wang** [comment]
the top bar menu seems not influenced
image

**Review by Chris-Wang — CHANGES_REQUESTED** [review]
Also the icon become smaller:
now:
image
the platform_app:
image

**Comment by Chris-Wang** [comment]
I doubt the Typograph PR has overwritten some styling here

**Comment by Chris-Wang** [comment]
does it overwrite the shadow as well?

**Comment by NorrisWu0** [comment]
wanna check again? @Chris-Wang

**Review by Chris-Wang — DISMISSED** [review]
Icon and label not align correctly 
image

**Comment by NorrisWu0** [comment]
> Icon and label not align correctly image

~~these seems identical to me, what's the difference?~~ right, I see, looks like fonts doesn't always have the same `line-height`, so much for doing the font scale work 🤷

**Comment by Chris-Wang** [comment]
it looks good now! just need resolve the conflict

**Review by s3713434 — APPROVED** [review]
933ef8584757a2a14c62e376f79fd474
Boom, approved.

**Pull request merged**